src/Microsoft.Data/StringDataFrameColumn.cs

```diff
@@ -426,7 +433,7 @@ public override GroupBy GroupBy(int columnIndex, DataFrame parent)
             }
         }
 
-        public override DataFrameColumn FillNulls(object value, bool inPlace = false)
+        public StringDataFrameColumn FillNulls(string value, bool inPlace = false)
         {
             if (value.GetType() != typeof(string) || value == null)
```

Comment:
no longer need `value.GetType() != typeof(string)`.

Especially because if `value` is null, this would have NRE.